.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,100 @@
+name: 🌈 Feature request
+description: Suggest an amazing new idea for this project
+title: "[FEATURE]:"
+labels:
+  - new feature
+  - requires triage
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Have a question?  This is the TypeORM issue tracker - and not the right
+        place for general support or questions. Instead, check the [“Support"
+        Documentation](https://github.com/typeorm/typeorm/blob/master/docs/support.md)
+        on the best places to ask questions!
+  - type: textarea
+    id: problem
+    attributes:
+      label: Feature Description
+      description: >-
+        A clear and concise description of what the problem is, and why it needs
+        to be implemented as a new feature for TypeORM.
+      placeholder: For example, "I'm always frustrated when ..."
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: The Solution
+      description: A clear and concise description of what you want to happen.
+      placeholder: >-
+        How can the requested feature be used to approach the problem it's
+        supposed to solve?
+    validations:
+      required: true
+  - type: textarea
+    id: alt
+    attributes:
+      label: Considered Alternatives
+      description: >-
+        A clear and concise description of any alternative solutions or features
+        you've considered.
+      placeholder: If any workaround exists to the best of your knowledge, include it here.
+    validations:
+      required: true
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional Context
+      description: Add any other context about the feature request here
+  - type: textarea
+    id: drivers
+    attributes:
+      label: Relevant Database Driver(s)
```

Comment:
better to use [checkboxes](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#checkboxes) instead.

Reply:
Done!